### DISCUSSION TITLE
74 153 ttl

### DISCUSSION THREAD

**Discussion by SBrbt** [logisim-evolution/logisim-evolution#1612] [General]
Chould u add the unit 74-153 ttl please ?
I like this software very much and use it in my university courses such as submitting digital logic circuit assignments to the teacher.  I have a request to add unit 74-153 ttl if possible.

**Comment by maehne** [comment]
A PR to add this component is welcome. You can have a look at the [implementation of similar TTL components](https://github.com/logisim-evolution/logisim-evolution/tree/master/src/main/java/com/cburch/logisim/std/ttl) to get you started.

@btb and @BunchOfBits contributed several TTL components overt the past year. Maybe, they are willing to assist you in this matter.

**Comment by BunchOfBits** [comment]
Done, see  PR #1613
